{"name": "Verify User is not able to Login with Invalid credentials", "status": "passed", "description": "Verify User is not able to Login with Invalid credentials", "attachments": [{"name": "log", "source": "737e136c-f70f-4630-bbf5-2fb544473cea-attachment.txt", "type": "text/plain"}], "start": 1589023145443, "stop": 1589023160685, "uuid": "038d1958-b6cf-44bd-b7ca-c1c5bf6d91ef", "historyId": "57f7ed73470ce8d4aacc4aec2dc80c63", "testCaseId": "e94f0024c374825736805ac54bd2db4c", "fullName": "tests.home.test_login.TestLogin#test_invalidLogin", "labels": [{"name": "severity", "value": "minor"}, {"name": "story", "value": "HCMAT-54574"}, {"name": "tag", "value": "HCMAT-54574"}, {"name": "parentSuite", "value": "tests.home"}, {"name": "suite", "value": "test_login"}, {"name": "subSuite", "value": "TestLogin"}, {"name": "host", "value": "del1-lhp-N01966"}, {"name": "thread", "value": "7008-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "tests.home.test_login"}]}